packages/core/src/util/json.ts

```diff
@@ -0,0 +1,78 @@
+const jsonify = (almostJson: string) => {
+  try {
+    return JSON.parse(almostJson)
+  } catch (e) {
+    // eslint-disable-next-line no-useless-escape
+    const tryFindJson = almostJson.replace(/([a-zA-Z0-9_$]+\s*):/g, '"$1":').replace(/'([^']+?)'([\s,\]\}])/g, '"$1"$2')
```

Comment:
This issue https://github.com/ckb-js/kuai/pull/301#discussion_r1242638807 still exists